tests/tests-asn1c-compiler/07-int-OK.asn1.-B

```diff
@@ -1,3 +1,9 @@
+/*
+SPDX-FileCopyrightText: 2020-present Open Networking Foundation <info@opennetworking.org>
```

Comment:
I think you can leave out the copyright - it's hard enough for the tool to compare outputs without adding this in
Add an exception to the license check if you must